manager/ui/apiman-plugin/plugins/apiman/html/user-services.html

```diff
@@ -7,46 +7,17 @@
   </head>
 
   <body>
-    
+  <body>
+    <div ng-controller="Apiman.UserServicesController">
     <!-- Top header/nav bar -->
-    <div id="apiman-header">
-      <div class="main-header container">
-        <div class="row">
-          <div class="col-md-12">
-            <div class="logo"></div>
-            <input class="search no-phone" type="text" placeholder="Quick service search..."></input>
-            <ul class="user-menu pull-right">
-              <li class="dropdown">
-                <a href="#" class="dropdown-toggle" data-toggle="dropdown">
-                  <span class="pficon pficon-user"></span>
-                  <span>Bruce Wayne</span>
-                  <b class="caret"></b>
-                </a>
-                <ul class="dropdown-menu">
-                  <li>
-                    <a href="user-orgs.html">Home</a>
-                  </li>
-                  <li class="divider"></li>
-                  <li>
-                    <a data-field="toProfile" href="settings-profile.html" data-i18n-key="profile">Profile</a>
-                  </li>
-                  <li>
-                    <a href="#">Logout</a>
-                  </li>
-                </ul>
-              </li>
-            </ul>
-          </div>
-        </div>
-      </div>
-    </div>
+    <div id="apiman-header" ng-include="'plugins/apiman/html/navbar.html'"></div>
```

Comment:
Before the navbar there should be a page loading progress indicator:

https://github.com/apiman/apiman/blob/master/manager/ui/apiman-plugin/plugins/apiman/html/org-plans.html#L11